discord/guild.py

```diff
@@ -3704,6 +3713,14 @@ async def create_role(
         colour: Union[:class:`Colour`, :class:`int`]
             The colour for the role. Defaults to :meth:`Colour.default`.
             This is aliased to ``color`` as well.
+        primary_color: Union[:class:`Colour`, :class:`int`, None]
```

Comment:
`color == primary_color`, not much point exposing both IMO
also missing the `_colour` aliases